### PR TITLE
Chart: update Multiple Axes demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Charts/MultipleAxes/description.md [hunk 1]
```diff
@@ -1 +1,11 @@
-With the Chart component, you can display charts with multiple axes and combine charts together to&nbsp;help better visualize the similarities or&nbsp;differences in&nbsp;data under investigation.
\ No newline at end of file
+With the Chart component, you can display charts with multiple axes for better data visualization. For example, in a single-axis chart, you have to choose which values to indicate - absolute or percentage. With a multi-axis chart, you can indicate both absolute and percentage values on two separate axes, as you can see in this demo.
+
+To configure a multi-axis chart, follow the steps below:
+
+1. Create and name the value axes.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/MultipleAxes/description.md]: We do need this extra emphasis here. Just create and name value axes.

```suggestion
1. Create and name value axes.
```

File: JSDemos/Demos/Charts/MultipleAxes/description.md [hunk 2]
```diff
@@ -1 +1,11 @@
-With the Chart component, you can display charts with multiple axes and combine charts together to&nbsp;help better visualize the similarities or&nbsp;differences in&nbsp;data under investigation.
\ No newline at end of file
+With the Chart component, you can display charts with multiple axes for better data visualization. For example, in a single-axis chart, you have to choose which values to indicate - absolute or percentage. With a multi-axis chart, you can indicate both absolute and percentage values on two separate axes, as you can see in this demo.
+
+To configure a multi-axis chart, follow the steps below:
+
+1. Create and name the value axes.         
+Declare multiple objects in the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/) array. Each object configures a single value axis. Then, give each value axis a unique [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name).
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/MultipleAxes/description.md]: A == one, that is, single. When you use both "a single", you are putting a very strong emphasis on singularity. Do we really need it? I have my doubts.

```suggestion
Declare multiple objects in the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/) array. Each object configures a value axis. Every value axis should have a unique [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name).
```

File: JSDemos/Demos/Charts/MultipleAxes/description.md [hunk 3]
```diff
@@ -1 +1,11 @@
...
+
+2. Bind series to value axes.        
+Assign the axis name to the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#axis) property. If you do not specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#axis) property, such a series is bound to the axis declared first in the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/) array.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/MultipleAxes/description.md]: To bind a series to **any** axis, we should assign the name of an axis. We need the name of any (target) axis.

```suggestion
Assign the name of an axis to the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#axis) property. If leave the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[axis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#axis) property unspecified, the series is bound to the axis declared first in the [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/) array.
```

File: JSDemos/Demos/Charts/MultipleAxes/description.md [hunk 4]
```diff
@@ -1 +1,11 @@
...
+
+In this demo, the first axis is declared without the [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name) and is bound to the first series. The second axis [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name) is `total` and it is bound to the spline series. The [position](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#position) of the second axis is `right`.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/MultipleAxes/description.md]: > The second axis [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name) is `total` and it is bound to the spline series.

There is a logical flaw. This sentence in its current state says that the mentioned name is bound to the spline series, not the axis. I do not think we can say that the axis and its name can be used interchangeably in terms of semantics.

```suggestion
In this demo, the first axis is declared without a [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name) and is bound to the first series. The [name](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#name) of the second axis is `total` and it is bound to the spline series. The [position](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/#position) of the second axis is `right`.
```

File: JSDemos/Demos/Charts/MultipleAxes/description.md [hunk 5]
```diff
@@ -1 +1,11 @@
-With the Chart component, you can display charts with multiple axes and combine charts together to&nbsp;help better visualize the similarities or&nbsp;differences in&nbsp;data under investigation.
\ No newline at end of file
+With the Chart component, you can display charts with multiple axes for better data visualization. For example, in a single-axis chart, you have to choose which values to indicate - absolute or percentage. With a multi-axis chart, you can indicate both absolute and percentage values on two separate axes, as you can see in this demo.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/Charts/MultipleAxes/description.md]: Could you please clarify why did you use the word "indicate" in this context? Don't you think the words like "designate" or "mark" would look better here?

Moreover, I can say that this isn't the best possible sentence...

> For example, in a single-axis chart, you have to choose which values to indicate - absolute or percentage.

It sounds too colloquial for our documentation.

Also, you start two sentences in this short paragraph with "With". It doesn't look good as well.

Can you rewrite this paragraph?

**Review by dmitry-eliseev-devexpress — COMMENTED** [review]
Think carefully about every word you use. You should be able to explain your choice - both to yourself and to anyone else. Once you'll be able to justify every single word in your text - it will become flawless.

**Review by dmitry-eliseev-devexpress — APPROVED** [review]

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**